Registry/Registry.EM_COMMON

```diff
@@ -397,6 +397,8 @@ state    real   Q2               ij     misc        1         -     irh01{22}{23
 state    real   T2               ij     misc        1         -     i01rh01{22}{23}du  "T2"                   "TEMP at 2 M"       "K"
 state    real   TH2              ij     misc        1         -     irhdu     "TH2"                  "POT TEMP at 2 M"   "K"
 state    real   PSFC             ij     misc        1         -     i01rhdu   "PSFC"                 "SFC PRESSURE"      "Pa"
+# new diag
```

Comment:
@thomasedds 
Thomas,
Remove these identifying pieces: "new new", "Thomas". I mention it here, but look at the rest of your PR mods, also.

Reply:
I find that RIP has this variable, and it has a unit of 'C'. So I think we should use 'C'.